hwconf/trampa/vesc4/hw_410_005ohm.h

```diff
@@ -0,0 +1,26 @@
+/*
+	Copyright 2015 Benjamin Vedder	benjamin@vedder.se
+
+	This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    */
```

Comment:
Outside the scope of this PR, but we really should get rid of this boiler plate from all of these header files. It has no legal validity, and it just gunks up text searches.